cspyce/__init__.py

```diff
@@ -180,8 +213,8 @@ def get_all_funcs(source=None, cspyce_dict=None):
     return cspyce_dict
 
 def get_all_versions(func, source=None):
-    """Return a dictionary of all cspyce functions associated with this one,
-    keyed by their names.
+    """A dictionary of all cspyce functions associated with this one, keyed by
```

Comment:
I'm not sure why you changed all these docstrings. The new ones don't agree with PEP257: 

```
The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.
```

Since this will be a public library, we should follow PEP257, including the requirement for the first line to be a single phrase that doesn't wrap to another line.

Reply:
I didn't touch these.  According to my git history, these were changed before I ever saw it.